doc/releases/v0.3.0.txt

```diff
@@ -1,6 +1,6 @@
 
-In Development: v0.3.0 (Unreleased)
------------------------------------
+In Development: v0.3.0
```

Comment:
Should we remove "In Development" as well, or are you waiting to do that until we actually tag?